### PR TITLE
Implement attributes wrapper for views

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/110022

@bordoni this is a WIP, not sure if I got you 100% on this one. Perhaps you can check it and we can discuss any further in a HO. Pushing this just for you to check the changes.

### CODE REVIEW

File: src/Tribe/Template.php [hunk 1]
```diff
@@ -33,5 +33,20 @@ public function attributes( $default_attributes = array() ) {
...
+
+	/**
+	 * Return a specific attribute
+	 *
+	 * @since TBD
+	 *
+	 * @param array $default_attributes
+	 * @return array
+	 */
+	public function attr( $index, $default_attributes = array() ) {
```
Comment (mitogh) [inline src/Tribe/Template.php]: Wonder if this just return a single attribute value we should specify the default value for that attribute only not the whole block

File: src/Tribe/Blocks/Abstract.php [hunk 2]
```diff
@@ -23,6 +23,19 @@ public function name() {
...
+	 * Return the block attributes
+	 *
+	 * @since  TBD
+	 *
+	 * @param  array $attributes
+	 *
+	 * @return array
+	*/
+	public function attributes( $params = array() ) {
+		return apply_filters( 'tribe_block_attributes_defaults_' . $this->slug(), $params );
```
Comment (bordoni) [inline src/Tribe/Blocks/Abstract.php]: Add docblock, also pass the `$this` variable. As the last param.
Lets name this one without the Slug, so users can target all blocks via one filter if needed.

File: src/Tribe/Template.php (hunk 1 above)
Comment (bordoni) [reply src/Tribe/Template.php]: Agree with this.

File: src/Tribe/Blocks/Abstract.php (hunk 2 above)
Comment (bordoni) [reply src/Tribe/Blocks/Abstract.php]: Both filters can be applied here, just make sure we create an abstract method for the `default_attributes` so we can setup em on each block.

**Comment by juanfra** [comment]
Thanks @mitogh and @bordoni! I just pushed some changes. Not totally sure if that's what you wanted @bordoni, but I think this gives us the flexibility to set the defaults per block and to make them hookable.

**Comment by mitogh** [comment]
Maybe we can remove this branch @juanfra ?

**Comment by juanfra** [comment]
Absolutely, sorry for that @mitogh 🎉

**Comment by mitogh** [comment]
🏆 @juanfra

**Pull request merged**